test/tools/router_check/router.cc

```diff
@@ -396,6 +402,14 @@ bool RouterCheckTool::compareResults(const std::string& actual, const std::strin
   return false;
 }
 
+bool RouterCheckTool::runtimeMock(const std::string& key, testing::Unused, testing::Unused) {
```

Comment:
nit: instead of `testing:Unused` I would put in the real parameters in case we use them later.

Reply:
I had to use `testing::Unused` to silence the parameter not used compiler warning.
It is not an issue anymore because we are now using the properties explicitly.